lib/msfl_visitors/renderers/chewy_renderer.rb

```diff
@@ -0,0 +1,41 @@
+module MSFLVisitors
+  module Renderers
+    class ChewyRenderer
+      BINARY_OPERATORS = {
```

Comment:
I just didn't realize how broad of a definition "anything" was in the context of hash keys.